scripts/accumulator-server.py

```diff
@@ -115,6 +114,18 @@ def all_done():
 
 app = Flask(__name__)
 
+@app.route("/v1/updateContext", methods=['POST'])
+def record():
+    global ac
+    ac += "Got a /v1/updateContext";
+    return Response(status=200)
+
+@app.route("/v1/queryContext", methods=['POST'])
+def record():
+    global ac
+    ac += "Got a /v1/queryContext";
+    return Response(status=200)
+
```

Comment:
What is even more weird is  having three times the same function defined, i.e. `record()`... maybe the `@app.route` is taken into account in the function definition and makes signatures different.

It seems that the function that is actually called is the last instance of `record()`. That would explain what it works and you don't have the "Got blabla" messages in the /dump.

Try to remove the first two function instances and set th following in the last one:

```
@app.route("/v1/updateContext", methods=['POST'])
@app.route("/v1/queryContext", methods=['POST'])
@app.route(server_url, methods=['GET', 'POST', 'PUT', 'DELETE'])
def record():
    global ac, t0, times
    ...
```